modules/installation-program-auto-create-subnets.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+
+[id="installation-program-auto-create-subnets_{context}"]
+= Setting the installation program to automatically create subnets
+
+// This feature is related to the edge compute pool only?
+
+You can set the installation program to automatically create public and private subnets for each Local Zone region that you defined in your installation configuration file. This automation also extends to the installation program running nodes in each Local Zone region that contains `machineset` manifest files.
+
+// What other functionality can the automation achieve?
```

Comment:
Would it be best to state all the actions that the automation completes on behalf of the user, so that a user does not accidentally skip certain configurations?

Reply:
Thanks, @mtulio . I included the list at line 14.